### PR TITLE
[fix](log) regularise some BE error type and fix a load task check

### PR DESCRIPTION
**Pull request by zclllyybb**
## Proposed changes

Issue Number: close #xxx

1. change some error message like
```
[INTERNAL_ERROR]The depth of the expression tree is too big, make it less than 600
```
to
```
[EXCEEDED_LIMIT]The depth of the expression tree is too big, make it less than 600
```
and with disable stacktrace

2. btw, fix a wrong disk limit checking in `EngineBatchLoadTask::_init()`
3. remove some BE config which is not used anymore.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zclllyybb** [comment]
run buildall

**Review by zhiqiang-hhhh — APPROVED** [review]
LGTM

File: be/src/common/status.h [hunk 1]
```diff
@@ -93,7 +93,7 @@ namespace ErrorCode {
     E(VERSION_NOT_EXIST, -214, false);                       \
     E(TABLE_NOT_FOUND, -215, true);                          \
     E(TRY_LOCK_FAILED, -216, false);                         \
-    E(OUT_OF_BOUND, -218, true);                             \
+    E(OUT_OF_BOUND, -218, false);                            \
```
Comment (gavinchou) [inline be/src/common/status.h:96]: Maybe a new error EXCEED_LIMIT is better?

**Comment by zclllyybb** [comment]
run buildall

**Comment by zclllyybb** [comment]
run buildall

**Comment by zclllyybb** [comment]
run buildall

**Comment by zclllyybb** [comment]
run buildall

**Comment by zclllyybb** [comment]
run buildall

File: be/src/olap/rowset/segment_v2/segment_writer.cpp [hunk 2]
```diff
@@ -1005,8 +1005,8 @@ Status SegmentWriter::finalize(uint64_t* segment_file_size, uint64_t* index_size
     timer.start();
     // check disk capacity
     if (_data_dir != nullptr && _data_dir->reach_capacity_limit((int64_t)estimate_segment_size())) {
-        return Status::Error<DISK_REACH_CAPACITY_LIMIT>("disk {} exceed capacity limit, path: {}",
```
Comment (yiguolei) [inline be/src/olap/rowset/segment_v2/segment_writer.cpp:1008]: 不要改

File: be/src/olap/task/engine_batch_load_task.cpp [hunk 3]
```diff
@@ -110,8 +110,10 @@ Status EngineBatchLoadTask::_init() {
 
     // check disk capacity
     if (_push_req.push_type == TPushType::LOAD_V2) {
-        if (tablet->data_dir()->reach_capacity_limit(_push_req.__isset.http_file_size)) {
-            return Status::IOError("Disk does not have enough capacity");
+        if (tablet->data_dir()->reach_capacity_limit(_push_req.http_file_size)) {
+            return Status::Error<EXCEEDED_LIMIT>(
+                    "reach the capacity limit of path {}, file_size={}", tablet->data_dir()->path(),
```
Comment (yiguolei) [inline be/src/olap/task/engine_batch_load_task.cpp:115]: not modify

File: be/src/olap/types.h [hunk 4]
```diff
@@ -1444,7 +1444,7 @@ struct FieldTypeTraits<FieldType::OLAP_FIELD_TYPE_STRING>
                               const int scale) {
         size_t value_len = scan_key.length();
         if (value_len > config::string_type_length_soft_limit_bytes) {
-            return Status::Error<ErrorCode::INVALID_ARGUMENT>(
+            return Status::Error<ErrorCode::EXCEEDED_LIMIT>(
```
Comment (yiguolei) [inline be/src/olap/types.h:1447]: not modify

File: be/src/vec/common/schema_util.cpp [hunk 5]
```diff
@@ -395,8 +395,8 @@ Status get_least_common_schema(const std::vector<TabletSchemaSPtr>& schemas,
     inherit_tablet_index(output_schema);
     if (check_schema_size &&
         output_schema->columns().size() > config::variant_max_merged_tablet_schema_size) {
-        return Status::DataQualityError("Reached max column size limit {}",
-                                        config::variant_max_merged_tablet_schema_size);
+        return Status::Error<ErrorCode::EXCEEDED_LIMIT>(
```
Comment (yiguolei) [inline be/src/vec/common/schema_util.cpp:398]: not modify

File: be/src/vec/olap/olap_data_convertor.cpp [hunk 6]
```diff
@@ -620,9 +620,9 @@ Status OlapBlockDataConvertor::OlapColumnDataConvertorVarChar::convert_to_olap(
                 slice->size = *offset_cur - string_offset;
                 if (UNLIKELY(slice->size > config::string_type_length_soft_limit_bytes &&
                              _check_length)) {
-                    return Status::NotSupported(
-                            "Not support string len over than "
-                            "`string_type_length_soft_limit_bytes` in vec engine.");
+                    return Status::Error<ErrorCode::EXCEEDED_LIMIT>(
```
Comment (yiguolei) [inline be/src/vec/olap/olap_data_convertor.cpp:623]: buyao dong

File: be/src/vec/olap/olap_data_convertor.cpp [hunk 7]
```diff
@@ -641,9 +641,9 @@ Status OlapBlockDataConvertor::OlapColumnDataConvertorVarChar::convert_to_olap(
             slice->size = *offset_cur - string_offset;
             if (UNLIKELY(slice->size > config::string_type_length_soft_limit_bytes &&
                          _check_length)) {
-                return Status::NotSupported(
-                        "Not support string len over than `string_type_length_soft_limit_bytes`"
-                        " in vec engine.");
+                return Status::Error<ErrorCode::EXCEEDED_LIMIT>(
+                        "length of string parameter is too long[len={}, max_len={}].", slice->size,
```
Comment (yiguolei) [inline be/src/vec/olap/olap_data_convertor.cpp:645]: andhere

**Comment by zclllyybb** [comment]
run buildall

**Review by yiguolei — DISMISSED** [review]
LGTM

**Comment by zclllyybb** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**